cmd/thanos/rule.go

```diff
@@ -564,6 +564,27 @@ func runRule(
 	return nil
 }
 
+// Scalar response consists of array with mixed types so it needs to be
+// unmarshaled separatelly.
+func convertScalarJSONToVector(scalarJSONResult json.RawMessage) (model.Vector, error) {
+	var resultPointSlice []json.RawMessage
```

Comment:
do you think we could do `[2]json.RawMessage`?

Reply:
Oh you answered my question sooner than I asked it in the comment below :)
Sorry I should have read on before I started commenting.